src/MetaCompilation.Analyzers/MetaCompilation.Analyzers.sarif

```diff
@@ -4,7 +4,10 @@
   "runs": [
     {
       "tool": {
-        "name": "Microsoft Code Analysis",
+        "name": "MetaCompilation.Analyzers",
+        "version": "42.42.42.42",
+        "fileVersion": "42.42.42.42424",
```

Comment:
Do we need to have all these versions?

Reply:
That's the same information the compiler emits. That's why u did it.

Reply:
Yes, but that is part of the build log. This version info is now going into files checked into the repo. Also how is this information useful to the viewer? It will just state that the file was generated using a debug assembly with version 42.42.42. Is that the intent?

Reply:
I noticed that.

Just out of curiosity, why don't the assemblies built have the "correct" version?

Reply:
These are debug assemblies, which are assigned the default 42.42.42 version by the arcade tooling.

I think given these SARIF files ship within our NuGet packages, the version information in them should match the version of the NuGet package and the packaged assembly versions. I would do the following to start with:
1. Only add "version" property to the log and avoid adding fileVersion and semanticVersion until we believe they can differ, which means they provide a value add, AND we can adjust our tooling or SARIF file generation to happen on each build so these reflect the versions of shipped assemblies, not just the versions of assemblies used to generate these files locally.
2. Use the version property "VersionPrefix" as the emitted version.